src/main/java/roomescape/domain/view/viewController.java

```diff
@@ -0,0 +1,18 @@
+package roomescape.domain.view;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class viewController {
```

Comment:
1.
```suggestion
@Controller
public class ViewController {
```

2. View의 의미는 무엇인가요?

Reply:
1. 대문자로 수정했습니다. 
2. 저는 여기서는 화면 (렌더링 된 Html)을 의미했어요 !

Reply:
1. 클래스명을 카멜케이스로 작성했네요 수정했습니다 !

2. 저는 여기서 화면(렌더링된 HTML)를 표현하기위해 View 를 사용했어요.

Reply:
[개인적인 의견] View는 여러가지 문맥에서 여러 관점으로 해석될 수 있어요. Spring MVC에서 View는 데이터가 보여지는 방식을 의미해요. 그래서 JSON 형식도 하나의 View라고 볼 수 있어요. 그래서 현재 객체의 역할을 표현하기에는 상당히 추상적이라고 볼 수 있을 것 같아요.

Reply:
좋은 관점 감사합니다. 한번더 생각해보고 다음 미션에 반영해볼게요 !